contracts/interfaces/IInterchainTokenService.sol

```diff
@@ -179,13 +179,14 @@ interface IInterchainTokenService is
     function registerTokenMetadata(address tokenAddress, uint256 gasValue) external payable;
 
     /**
-     * @notice Links a source token to a destination token on a remote chain.
-     * @param salt The salt used for token manager deployment.
-     * @param destinationChain The name of the destination chain.
-     * @param destinationTokenAddress The address of the token on the destination chain.
-     * @param tokenManagerType The type of token manager. Cannot be NATIVE_INTERCHAIN_TOKEN.
-     * @param linkParams The link parameters.
-     * @param gasValue The gas value for deployment.
+     * @notice This replaces the old deployTokenManager function.
```

Comment:
Should describe what the method does first and then add a note on the deployTokenManager method (what a new user to ITS will want to know about)